src/main/java/seedu/duke/commands/ExitCommand.java

```diff
@@ -6,15 +6,15 @@
  * Displays goodbye message when application is terminated by the user.
  */
 public class ExitCommand extends Command {
-    public void executeCommand(ProjectList projects) {
-        printGoodbyeMessage();
+    public String executeCommand(ProjectList projects) {
+        return printGoodbyeMessage();
     }
 
     public Boolean isExit() {
         return true;
     }
 
-    private void printGoodbyeMessage() {
-        System.out.println("See you again!");
+    private String printGoodbyeMessage() {
+        return "See you again!";
     }
```

Comment:
Would extracting to the UI class be neater? 
So everything related to printing will be under user interface.